apps/omg_api/test/support/state/prop_test/form_block.ex

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.API.State.PropTest.FormBlock do
```

Comment:
see the other comment on ` = block` - I was surprised there weren't a unused variable error.

The reason is that the body of the `post` is somehow skipped on this check. To fix this I propose https://github.com/omisego/elixir-omg/commit/add14554a14acfd26d3e8bbf5b95345bfbd8a496 as solution.